_component_design/key-value-pairs.md

```diff
@@ -1,40 +1,120 @@
 ---
-title: Key Value Pairs
-parent: unknown
+title: Key-value pairs
+parent: Content Areas
 layout: component
 category: Components
 usage: >
-  Key value pairs are used to display data with their corresponding label. Color
-  and alignment are used to enhance readability allowing users to quickly scan
-  for the page for relevant data.
-preview-image: null
```

Comment:
update to the following - `preview-image: preview-images/key-value.svg`

Reply:
I built a preview svg and placed it on the CDN this AM. If interested found here - http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/preview-images/key-value.svg

Reply:
updating to the first comment in this thread will render the following
![image](https://user-images.githubusercontent.com/586682/32572965-fd677c8e-c491-11e7-8515-36e0a9f45bea.png)